Networking/NetworkingTests/Remote/StripeRemoteTests.swift

```diff
@@ -0,0 +1,309 @@
+import XCTest
+@testable import Networking
+
+
+/// Stripe Remote Unit Tests
+///
+final class StripeRemoteTests: XCTestCase {
+
+    /// Dummy Network Wrapper
+    ///
+    let network = MockNetwork()
+
+    /// Dummy Site ID
+    ///
+    let sampleSiteID: Int64 = 1234
+
+    /// Repeat always!
+    ///
+    override func setUp() {
+        network.removeAllSimulatedResponses()
+    }
+
+    /// Verifies that loadAccount properly handles the nominal response. We'll also validate the
+    /// statement descriptor, currencies and country here.
+    ///
+    func test_loadAccount_properly_returns_account() throws {
+        let remote = StripeRemote(network: network)
+
+        network.simulateResponse(requestUrlSuffix: "wc_stripe/account/summary", filename: "stripe-account-complete")
+
+        let result: Result<StripeAccount, Error> = waitFor { promise in
+            remote.loadAccount(for: self.sampleSiteID) { result in
+                promise(result)
+            }
+        }
+
+        XCTAssertTrue(result.isSuccess)
+        let account = try result.get()
+        XCTAssertEqual(account.status, .complete)
+        XCTAssertEqual(account.statementDescriptor, "MY.FANCY.US.STORE")
+        XCTAssertEqual(account.defaultCurrency, "usd")
+        XCTAssertEqual(account.supportedCurrencies, ["usd"])
+        XCTAssertEqual(account.country, "US")
+        XCTAssertEqual(account.isCardPresentEligible, true)
```

Comment:
nit: might be nice to add the `Given/When/Then` comments like in other tests in the codebase, so that it's easy to understand each part 🙂 (same for other test cases)
```suggestion
        // Given
        let remote = StripeRemote(network: network)

        network.simulateResponse(requestUrlSuffix: "wc_stripe/account/summary", filename: "stripe-account-complete")

        // When
        let result: Result<StripeAccount, Error> = waitFor { promise in
            remote.loadAccount(for: self.sampleSiteID) { result in
                promise(result)
            }
        }

        // Then
        XCTAssertTrue(result.isSuccess)
        let account = try result.get()
        XCTAssertEqual(account.status, .complete)
        XCTAssertEqual(account.statementDescriptor, "MY.FANCY.US.STORE")
        XCTAssertEqual(account.defaultCurrency, "usd")
        XCTAssertEqual(account.supportedCurrencies, ["usd"])
        XCTAssertEqual(account.country, "US")
        XCTAssertEqual(account.isCardPresentEligible, true)
```

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/5799/commits/fa94cc58f6f49b527491f9d1eb1785429720be2f